ClangPowerTools/ClangPowerTools/MVVM/Command/RelayCommand.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Input;
+
+namespace ClangPowerTools.MVVM.Command
+{
+  public class RelayCommand : ICommand
+  {
+    public event EventHandler CanExecuteChanged
+    {
+      add { CommandManager.RequerySuggested += value; }
+      remove { CommandManager.RequerySuggested -= value; }
+    }
+
+    private readonly Action _action;
```

Comment:
Please replace the `_` with the new standard notation for class members. As far as I know the standard naming for `actions` is using upper case for the first letter.

Reply:
Ok, I'll remove the '_' from '_action'. That variable is private, so I'll keep the lower case for the first letter.